app.js

```diff
@@ -0,0 +1,107 @@
+"use strict";
+
+var restify = require('restify');
+var Sequelize = require('sequelize');
+
+var sequelize = new Sequelize('mvctodo', 'postgres', 'nicopostgres', {
+    host: "localhost",
+    port: 5432,
+    dialect: 'postgres'
+});
+
+var todoentries = sequelize.define('todoentries', {
+    title: {
+        type: Sequelize.STRING,
+        field: 'title'
+    },
+    completed: { type: Sequelize.BOOLEAN }
+},
+    {
+        createdAt: false,
+        updatedAt: false,
+        freezeTableName: true
+    }
+    );
+
+function CreateTodo(req, res, next) {
+    todoentries.sync({ force: true }).then(function () {
```

Comment:
You should not be syncing, this is something that you do once to change structure, not every request